lib/vanagon/platform.rb

```diff
@@ -239,7 +239,11 @@ def [](key)
     #   at the project level
     # @return [String] relative path to where packages should be output to
     def output_dir(target_repo = "")
-      @output_dir ||= File.join(@os_name, @os_version, target_repo, @architecture)
+      if target_repo == "PC1"
+        @output_dir ||= File.join(@os_name, @os_version, target_repo, @architecture)
+      else
+        @output_dir ||= File.join(target_repo, @os_name, @os_version, @architecture)
+      end
```

Comment:
yeah, I know it's not ideal. Haven't thought of a better solution yet, but also haven't given it much thought.